sgx-hasher/src/lib.rs

```diff
@@ -79,6 +79,55 @@ impl Hasher {
     }
 }
 
+fn bn_to_rsanum_inner(bn: bn::BigNum) -> Result<[u8; 384], openssl::error::ErrorStack> {
+    let mut array = [0u8; 384];
+    array.copy_from_slice(&bn.to_vec());
+    array.reverse();
+    Ok(array)
+}
+
+fn bn_to_u32(value: &bn::BigNum) -> Result<u32, openssl::error::ErrorStack> {
+    let mut buf = u32::default().to_ne_bytes(); // Zero
+    value
+        .to_slice(&mut buf)
+        .or(Err(openssl::error::ErrorStack::get()))?;
```

Comment:
1. I don't think that `ErrorStack::get()` works in this context because no openssl error was actually generated during `to_slice()`. So we need something different here. Maybe `std::io::Error` is the right error code (I know I told you to change it)?

2. Change `to_slice()` to `to_vec()` and copy the bytes around. Hopefully upstream will accept a version of `to_slice()` in the future.